srcpkgs/howl/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'howl'
+pkgname=howl
+version=0.6
+revision=1
+archs="i686 x86_64 x86_64-musl"
+build_wrksrc=src
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="wget gcc pkg-config gtk+3"
+makedepends="gtk+3-devel"
+depends="gtk+3"
```

Comment:
Remove the `depends` variable as xbps-src detects the runtime library by itself. `depends` can be set in the case xbps can't automatically identify the needed runtime libs.

Reply:
Right, omit this line entirely.

Reply:
Done.